Kickstarter-iOS/Views/Controllers/PledgeViewController.swift

```diff
@@ -75,8 +78,14 @@ final class PledgeViewController: UIViewController, MessageBannerViewControllerP
       |> \.delegate .~ self
   }()
 
+  private lazy var submitButton: UIButton = { UIButton(type: .custom) }()
```

Comment:
Is `init with type` the designated initializer?

Reply:
Yeah we want this to be a `custom` button to not have the highlight animation that `system` buttons get.